pkg/collector/corechecks/cluster/kubernetes_state.go

```diff
@@ -339,65 +328,26 @@ func (k *KSMCheck) metricFilter(m ksmstore.DDMetric) bool {
 	return m.Val == float64(1)
 }
 
-// isMatching returns whether a targeted metric for label joins is
-// matching another metric labels based on the labels to match config
-func isMatching(config *JoinsConfig, destlabels, srcLabels map[string]string) bool {
-	for _, l := range config.LabelsToMatch {
-		firstVal, found := destlabels[l]
-		if !found {
-			return false
-		}
-		secondVal, found := srcLabels[l]
-		if !found {
-			return false
-		}
-		if firstVal != secondVal {
-			return false
-		}
-	}
-	return true
-}
-
 // buildTag applies the LabelsMapper config and returns the tag in a key:value string format
 // The second return value is the hostname of the metric if a 'node' or 'host' tag is found, empty string otherwise
 func (k *KSMCheck) buildTag(key, value string) (tag, hostname string) {
 	if newKey, found := k.instance.LabelsMapper[key]; found {
```

Comment:
This method is doing three separate things here, it's actually hard to understand why it's called `buildTag`, returning `hostname` is quite confusing given the name.

Reply:
I see your point.
But I’m lacking inspiration.
What would be a good function name, explicit but without being outrageously too verbose, for a function that turns a KSM label key value pair into a Datadog tag?